app/v1/controllers/requestParsers/validators/validations/ValueFormatErrorsMessages.scala

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.controllers.requestParsers.validators.validations
+
+protected[validators] trait ValueFormatErrorMessages {
+  val ZERO_MINIMUM_INCLUSIVE = "The field should be between 0 and 99999999999.99"
```

Comment:
Can this be updated to `The field should be between 0.00 and 99999999999.99` match the netVatDue description in the doc. Thanks